shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -257,6 +253,25 @@
 }
 
 void FlutterPlatformViewsController::DetachUnusedLayers() {
+  // Clean up disposed views.
+  if (!views_waiting_to_dispose_.empty()) {
+    for (std::vector<int64_t>::iterator it = active_composition_order_.begin(); it != active_composition_order_.end();) {
```

Comment:
we should dispose the view even if it's not in `active_composition_order_`

Reply:
Make sense, I will fix it on Monday.

Reply:
Done!